devices/xiaomi.js

```diff
@@ -1486,7 +1843,49 @@ module.exports = [
             exposes.enum('motion_sensitivity', ea.ALL, ['low', 'medium', 'high']).withDescription('Different sensitivities ' +
                 'means different static human body recognition rate and response speed of occupied'),
             exposes.enum('reset_nopresence_status', ea.SET, ['']).withDescription('Reset the status of no presence'),
-            e.device_temperature(), e.power_outage_count()],
+            e.device_temperature(), e.power_outage_count(),
+            exposes.enum('region_event', ea.STATE, ['region_*_enter', 'region_*_leave', 'region_*_occupied',
```

Comment:
I suggest to rename this to `action`, then this can also be easily in Home Assistant for automations.

Reply:
Makes sense, however I'm wondering if `presence_event` should be merged into that property as well? That's probably something for another PR, and should most likely be introduced in a non-breaking manner (eg. first merge these two, deprecate `presence_event`, and then remove it in next release), but I could work on that (later) as well, if that makes sense for you?

Reply:
Good idea, please expose the `presence_event` as the `action` also. (and keep the `presence_event`). Can also be done in a later PR.